common/test/test_sshtools.py

```diff
@@ -276,13 +276,6 @@ def test_sshHostKey(self):
 
         hostKey = '/etc/ssh/ssh_host_{}_key.pub'.format(keyType.lower())
         self.assertExists(hostKey)
-        self.assertEqual(3, len(keyHash.split()))
```

Comment:
This does not work when it is an ed25519 key instead of rsa.
I also don't see much value in testing content of a key file. So I removed it without replacement.